script/libtest.sh

```diff
@@ -45,11 +45,20 @@ k8s_wait_for_pod_completed() {
     k8s_wait_for_pod $namespace $labelSelector Completed
 }
 
+k8s_ensure_image() {
+    namespace=${1:?}
+    deployment=${2:?}
+    expectedPattern=${3:?}
+    jsonpath=${4:-'{.spec.template.spec.containers[0].image}'}
+    echo "Checking image for $deployment"
```

Comment:
Could we make this more descriptive.

Something like, "checking that deployment %s is using image tagged with %s"

It would be ideal to get a message when it is not the case as well but anyways, not important.

Reply:
note that the `jsonpath` is an optional parameter, I thought that this can be reused for a different thing, not only the image (unless we are not probably going to use it with something different)